internal/commands/ssh.go

```diff
@@ -38,6 +38,7 @@ func (c *SSHCommand) Run(s *settings.Settings) {
 	args := []string{
 		paths.GetCmdPathOrExit("ssh"),
 		"-i", paths.GetDataFilePath(constants.SSHKeypairName),
+		"-S", "none",
```

Comment:
The point is I got an 
```Host *
    ControlMaster auto
    ControlPath ~/.ssh/ssh-%r@%n:%p
```
and  I want to disable controlsocket for hackenv, to prevent controlsocket problem that happen sometimes - ControlSocket often help a lot but can be buggy from my experience, in hackenv for the few ssh connections we do, controlsocket seems useless so I prefer explicitly disable it. Make sense?